network/benchmarks/README.md

```diff
@@ -79,9 +79,15 @@ Here is a simple all-reduce benchmark that you can use to quickly measure the th
 
 [all_reduce_bench.py](all_reduce_bench.py)
 
+On CSPs that have enabled [SLURM Pyxis Container Plugin](https://github.com/NVIDIA/pyxis), such as CoreWeave, Crusoe, AWS, Oracle, Azure, GCP, etc, `all_reduce_bench.py` can be easily ran & reproduced via the following command:
+```bash
+sbatch -n <num_of_nodes> ./all_reduce_bench_pyxis.sbatch
```

Comment:
you probably want to remove `-n <num_of_nodes>` because your batch script will break if someone sets anything but `-n 2`.

Reply:
your probably can automate and not hardcode `NNODES=` entry in the bash script by deriving it from the slurm environment and then it'd work.

Reply:
(after your edit) - that's better ;)

Reply:
thanks @stas00 for the suggestion.

I just changed it to follow your [launcher example here](https://github.com/stas00/ml-engineering/blob/6524f463a3d5c9a0a311bb82feca0f196dc3141a/orchestration/slurm/launchers/torchrun-launcher.slurm#L32C8-L32C21)